docker_files/recipes/python/meta/README.md

```diff
@@ -0,0 +1,33 @@
+# Python 2.7
+This package provides core Python 2.7 runtime environment. It includes default builtin modules.
+
+## Usage
+Run Python shell:
+```
+$ capstan run python --boot shell
+```
+This will launchd default Python shell. To run a script, ensure the script is composed into the OSv
+unikernel and then run it using
+```
+$ capstan run python -e "--env=PYTHONHOME=/pyenv /python script.py"
```

Comment:
See my comment at meta/run.yaml. I would prefer to do it like this:
```bash
$ capstan run demo --boot python --env ARGS=script.py
```
Also please use `demo` for the unikernel name to be consistent with other READMEs.